services.yaml

```diff
@@ -3069,6 +3151,123 @@ definitions:
         items:
           type: string
           example: MyApp
+  Alert:
+    description: All alert settings of a view
+    type: object
+    properties:
+      alert_on_new: 
+        type: "boolean"
+        example: true
+      slack_settings: 
+        properties: 
+          enabled: 
+            type: "boolean"
+            example: true
+          inhook_url: 
+            type: "string"
+            example: "https://hooks.slack.com/services/xxxxyyyyy/xxxx/sdf123rskfsdf1j"
+        type: "object"
+      hip_chat_settings: 
+        properties: 
+          enabled: 
+            type: "boolean"
+            example: false
+          token: 
+            type: "string"
+            example: "dfgdk2342kljsldfjslk2342fsdfkjslkj"
+          room: 
+            type: "string"
+            example: "Takipi"
+        type: "object"
+      pager_duty_settings: 
+        properties: 
+          enabled: 
+            type: "boolean"
+            example: false
+          service_integration_key:
+            type: "string"
+            example: "sflk234lkdjflsj12o8312"
+        type: "object"
+      #jira_settings: <- TBD
+      #  properties: 
+      #    
+      #  type: "object"
+      webhook_settings: 
+        properties: 
+          enabled: 
+            type: "boolean"
+            example: false
+          webhook_url:
+            type: "string"
+            example: "http://hooks.company.com/overops1/P1/hook"
+        type: "object"
+      service_now_settings: 
+        properties: 
+          enabled: 
+            type: "boolean"
+            example: false
+          url:
+            type: "string"
+            example: "http://servicenow.company.com/hooks"
+          user_id:
+            type: "string"
+            example: "admin"
+          password: 
+            type: "string"
+            example: "pa$$1"
+          table:
+            type: "string"
+            example: "overops"
+        type: "object"
+      email_settings: 
+        properties: 
+          is_active: 
+            type: "boolean"
+            example: true
+          alert_me: 
+            type: "boolean"
+            example: true
+          alert_all_team_members: 
+            type: "boolean"
+            example: false
+          alert_additional_emails: 
+            type: "boolean"
+            example: false
+          additional_emails_to_alert:
+            type: "array"
+            items:
+              type: string
+              example: "user1@company.com"
+        type: "object"
+      udf_settings: 
+        properties: 
+          enabled: 
+            type: "boolean"
+            example: false
+          library_id: 
+            type: "string"
+            example: "L24"
+          function_id: 
+            type: "string"
+            example: "apply-label"
+          args:
+            type: "string"
+            example: "label=sample_label_name"
+        type: "object"
+      periodic_udf_settings: 
```

Comment:
I would move it to the beginning of the json (right after the 'alert-on-new' field) so it will be clear and consistent with the ui dialog. also, the 'enable' field is missing in the struct.